TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Positioning/PositionSystem.java

```diff
@@ -0,0 +1,79 @@
+package org.firstinspires.ftc.teamcode.Positioning;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import org.firstinspires.ftc.teamcode.Components.DriveTrain;
+import org.firstinspires.ftc.teamcode.Components.Gyro;
+import org.firstinspires.ftc.teamcode.Positioning.Utils.Vector2d;
+
+@Deprecated //INCOMPLETE AND UNSAFE!!!
+public class PositionSystem {
+
+    public enum TgtPositions{
+        DEPOT(new Vector2d(0,0)), CRATER(new Vector2d(0,0));
+
+        private final Vector2d tgtPosition;
+        TgtPositions(Vector2d id) { this.tgtPosition = id; }
+        public Vector2d getValue() { return tgtPosition; }
+    }
+
+    OpMode opMode;
+    DriveTrain driveTrain;
+    Gyro gyro;
+
+    private final double inchPerTick = 0.0D; //TODO: GET RATIO
+    private double oldSec;
+    private double oldDistance;
+    private double oldHeading;
+    private Vector2d position;
+
+    public PositionSystem(OpMode opMode, Gyro gyro, DriveTrain driveTrain){
+        this.opMode = opMode;
+        this.gyro = gyro;
+        this.driveTrain = driveTrain;
+    }
+
+    public void init(boolean debug){
+        if(debug){
+            opMode.telemetry.addData("INFO", "Pos Sys INIT");
+        }
+    }
+
+    public void updatePos(){
+        double pastSec = System.currentTimeMillis()/1000.0-oldSec;
+        if(pastSec > 0.25) {
+            double distance = ((driveTrain.getTicksBL() * inchPerTick +
```

Comment:
Given the test case where only one side is moving,
rightMovement = 0
leftMovement = 1000

totalDistance = 1000 *ipt/2, where the actuall distance could have been 0 if it went in a complete circle

(this is assuming that the robot is able to make a full rotation in .25 seconds, which it can't, but the problem still exists)

Reply:
We tried it in FRC testing at full speed turn, it did not come up as an issue, as addressed above regarding the .25 seconds update speed on position, it is 1/16 of a second and even as speedy as our robot is currently it does not have the ability to move at a speed that causes such an issue within 1/16 of a second

Reply:
okay, thanks for the clarification, I would still watch out for it if we ever run into problems. I think the general math here is a bit off though, I haven't fully ran through it in my head, but this isn't a huge issue ATM